tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -1880,11 +1838,53 @@ tensorflow::Status ConvertFusedBatchNorm(
   TRT_ShapedWeights mean_weights = inputs.at(3).weights();
   TRT_ShapedWeights variance_weights = inputs.at(4).weights();
   TRT_ShapedWeights dummy_power_weights(scale_weights.type_);
+  size_t scale_weights_count = scale_weights.count();
```

Comment:
Can you use for loops to reduce the boilerplate code from line 1836 to 1879?

Reply:
Originally avoiding for loop for readability. (scale, offset, mean, variance)
Will update them and put in comment.